helm/management-cluster-admission/Chart.yaml

```diff
@@ -5,4 +5,5 @@ home: https://github.com/giantswarm/management-cluster-admission
 version: [[ .Version ]]
 appVersion: '[[ .AppVersion ]]'
 annotations:
+  application.giantswarm.io/team: "biscuit"
```

Comment:
Not sure if you want to include that in the changelog because it's a reason to release IMO.